scripts/cat/age.py

```diff
@@ -0,0 +1,20 @@
+from enum import IntEnum
+
+
+class Age(IntEnum):
```

Comment:
Why are you using an IntEnum instead of a StrEnum? Is this intentional?

Reply:
It was intentional, yes - it has the very tiny win of letting you sort much easier lol. I had originally been using it for the logic stuff (in that you could go "if younger than adult" for example), but I've now put in a bunch of helpers to make the code easier to read for beginners (and also avoid having to import Age everywhere!)